assets/templates/datasetLandingPage/filterable.tmpl

```diff
@@ -54,7 +54,7 @@
                 <ul class="list--neutral">
                 {{range $i, $download := $v.Downloads}}
                 {{if gt (len $download.Size) 0}}{{if gt (len $download.Size) 0}}<li class="padding-left--1 margin-top--0 margin-bottom--1 white-background clearfix"><span class="inline-block padding-top--2">{{if eq $download.Extension "txt"}}Supporting information{{else}}Complete dataset (<span class="uppercase">{{$download.Extension}}</span> format){{end}}</span>
-                  <div class="width--12 inline-block float-right text-right"><a id="{{$download.Extension}}-download"                                     data-gtm-download-file="{{$download.URI}}"
+                  <div class="width--12 inline-block float-right text-right float-el--left-sm text-left"><a id="{{$download.Extension}}-download"                                     data-gtm-download-file="{{$download.URI}}"
```

Comment:
Sorry, I just realised we now have `text-right` and `text-left` classes being applied to the element, and the `text-left` rules would ultimately be the ones applied. We would want to tidy up this markup. I can see the mobile viewport looks like how we would want it to, while there is a slight change in non-mobile where the green button has moved slightly to the left 

I don't think it's a problematic visual change on `md` and `lg`, so it might be okay to remove `text-right` altogether. That said, it may be worth double checking to see if there is a modifier in the utility classes we could use to make it so that `text-left` rules only get applied on `sm` viewports

Reply:
You're right, sorry about that. I'll remove the `text-right`. Looking at it again, there's an issue in the mobile view for the Pixel 2 that I need to fix as well.

Reply:
Thanks - give me a shout if you need anything :)